composer.json

```diff
@@ -247,22 +244,23 @@
         "fakerphp/faker": "^1.16.0",
         "filp/whoops": "^2.7",
         "myclabs/deep-copy": "^1.8.1",
+        "phpstan/phpstan": "^1.9.2",
         "phpunit/phpunit": "^9.5.2",
-        "roave/better-reflection": "~3.3.0 || ~4.3.0",
-        "roave/signature": "1.2.0",
-        "sebastian/diff": "^3.0 || ^4.0",
         "sebastian/git": "^2.1",
         "sebastian/object-enumerator": "^3.0.3 || ^4.0",
         "sebastian/recursion-context": "^3.0.0 || ^4.0",
-        "sllh/composer-versions-check": "^2.0.2",
-        "spryker-feature/development-tools": "^202204.0",
+        "sllh/composer-versions-check": "^2.0.5",
+        "spryker-feature/development-tools": "^202211.0",
+        "spryker-sdk/benchmark": "0.2.2",
+        "spryker-sdk/composer-constrainer": "0.2.6",
```

Comment:
In case of removal composer will update several modules:
```
  - Upgrading nikic/php-parser (v4.15.1 => v4.15.2)
  - Upgrading composer/pcre (3.0.2 => 3.1.0)
  - Upgrading nette/schema (v1.2.2 => v1.2.3)
  - Upgrading symfony/polyfill-php81 (v1.26.0 => v1.27.0)
  - Upgrading symfony/polyfill-php72 (v1.26.0 => v1.27.0)
  - Upgrading symfony/polyfill-intl-idn (v1.26.0 => v1.27.0)
  - Upgrading symfony/polyfill-intl-icu (v1.26.0 => v1.27.0)
  - Upgrading laminas/laminas-filter (2.24.0 => 2.28.1)
  - Upgrading spryker/container (1.4.5 => 1.4.6)
  - Upgrading spryker/twig (3.17.1 => 3.18.0)
  - Upgrading spryker/transfer (3.33.0 => 3.33.1)
  - Upgrading symfony/polyfill-iconv (v1.26.0 => v1.27.0)
  - Upgrading spryker/gui (3.45.1 => 3.45.2)
  - Upgrading spryker/queue (1.9.2 => 1.9.3)
  - Upgrading spryker/propel (3.35.0 => 3.35.1)
  - Upgrading spryker/zed-request (3.18.0 => 3.18.1)
  - Upgrading spryker/router (1.14.0 => 1.14.1)
  - Upgrading spryker/storage (3.19.0 => 3.19.1)
  - Upgrading spryker/mail (4.9.0 => 4.10.0)
  - Upgrading spryker/customer (7.47.0 => 7.48.0)
  - Upgrading spryker/application (3.30.0 => 3.30.1)
  - Upgrading spryker/oms (11.21.0 => 11.22.1)
  - Upgrading spryker/search (8.19.0 => 8.19.1)
  - Upgrading spryker/collector (6.7.0 => 6.7.1)
  - Upgrading spryker/product-category (4.16.0 => 4.17.0)
  - Upgrading spryker/newsletter (4.5.2 => 4.6.0)
```
All demoshops should have the same versions for `spryker` and `spryker-shop` modules...
If you approve such updating - I will do removal.
cc: @DmitryLymarenko